src/components/dac_dataset_table/DACDatasetApprovalStatus.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import {DAC} from '../../libs/ajax';
+import {Link} from 'react-router-dom';
+import {isNil} from 'lodash/fp';
+import Button from '@mui/material/Button';
+import ReactTooltip from 'react-tooltip';
+import style from '../../pages/DACDatasets.module.css';
+
+export default function DACDatasetApprovalStatus(props) {
+
+  const [dataset, setDataset] = useState(props.dataset);
+
+  const updateApprovalStatus = async (approvalState) => {
+    const updatedDataset = await DAC.updateApprovalStatus(dataset.dacId, dataset.dataSetId, approvalState);
+    setDataset(updatedDataset);
+  };
+
+  const dacAccepted = (dataset) => <div style={{color: '#1ea371', fontWeight: 'bold'}}>
```

Comment:
Are the colors not part of a highlevel theme?

Reply:
These colors are not, as far as I know. They seem very different from the rest of the site for some reason.